python/giotto_json_reader.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+
+
+def cells_json(data):
+    '''
+    >>> data = {
+    ...     "cell_1": {
+    ...         "locations": [1632.02, -1305.7],
+    ...         "factors": {
+    ...             "pleiden_clus": [3],
+    ...             "kmeans": [8]
+    ...         },
+    ...         "mappings": {
+    ...             "tsne": [-11.0776, 6.0311],
+    ...             "umap": [-6.858, 15.7691]
+    ...         }
+    ...     },
+    ...     "cell_2": {
```

Comment:
(Since the end-to-end test also covers this, I'd keep the doc test as simple as possible: If we have tests inline, we need to be sure to keep them concise.)